advanced-api/dynamic-remotes-runtime-environment-variables/e2e/checkDynamicRemotesRuntimesApps.cy.ts

```diff
@@ -0,0 +1,104 @@
+import {BaseMethods} from "../../../cypress/common/base";
+import {baseSelectors} from "../../../cypress/common/selectors";
+import {Constants} from "../../../cypress/fixtures/constants";
+import {getDateWithFormat} from "../../../cypress/helpers/base-helper";
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        header: Constants.elementsText.dynamicSystemRemotesRuntime.host.header,
+        subheader: Constants.elementsText.dynamicSystemRemotesRuntime.host.subHeader,
+        hostH3: Constants.elementsText.dynamicSystemRemotesRuntime.host.hostH3,
+        paragraph: Constants.elementsText.dynamicSystemRemotesRuntime.paragraph,
+        button: Constants.elementsText.dynamicSystemRemotesRuntime.host.button,
+        loading: Constants.elementsText.dynamicSystemRemotesRuntime.loading,
+        buttonHeader: Constants.elementsText.dynamicSystemRemotesRuntime.buttonHeader,
+        buttonH2: Constants.elementsText.dynamicSystemRemotesRuntime.buttonH2,
+        buttonParagraph: Constants.elementsText.dynamicSystemRemotesRuntime.buttonParagraph,
+        host: 3000
+    },
+
+    {
+        header: Constants.elementsText.dynamicSystemRemotesRuntime.host.header,
+        subheader: Constants.elementsText.dynamicSystemRemotesRuntime.remote.subHeader,
+        loading: Constants.elementsText.dynamicSystemRemotesRuntime.loading,
+        buttonHeader: Constants.elementsText.dynamicSystemRemotesRuntime.buttonHeader,
+        buttonH2: Constants.elementsText.dynamicSystemRemotesRuntime.buttonH2,
+        buttonParagraph: Constants.elementsText.dynamicSystemRemotesRuntime.buttonParagraph,
+        host: 3001
+    }
+]
+
+appsData.forEach((
+    property: {
+        header: string,
+        subheader: string,
+        hostH3?: string,
+        paragraph?: string,
+        button?: string,
+        loading: string,
+        buttonHeader: string,
+        buttonH2?: string,
+        buttonParagraph: string,
+        host: number
+    }
+) => {
+    describe(`Check Host and Remote apps`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+        it (`Check host and Remote apps starts and running`, () => {
```

Comment:
You can use here `property.subHeader`
Update please to use it `Check ${property.subheader} app start and running + check app elements`